tests/jax/test_image_wrapping.py

```diff
@@ -0,0 +1,150 @@
+import jax
+import jax_galsim as galsim
+import numpy as np
+from galsim_test_helpers import timer
+
+from jax_galsim.core.wrap_image import (
+    expand_hermitian_x,
+    expand_hermitian_y,
+    contract_hermitian_x,
+    contract_hermitian_y,
+)
+
+
+@timer
+def test_image_wrapping_expand_contract():
+    # For complex images (in particular k-space images), we often want the image to be implicitly
+    # Hermitian, so we only need to keep around half of it.
+    M = 38
+    N = 25
+    K = 8
+    L = 5
+    im = galsim.ImageCD(2 * M + 1, 2 * N + 1, xmin=-M, ymin=-N)  # Explicitly Hermitian
+    im2 = galsim.ImageCD(
+        2 * M + 1, N + 1, xmin=-M, ymin=0
+    )  # Implicitly Hermitian across y axis
+    im3 = galsim.ImageCD(
+        M + 1, 2 * N + 1, xmin=0, ymin=-N
+    )  # Implicitly Hermitian across x axis
+    # print('im = ',im)
+    # print('im2 = ',im2)
+    # print('im3 = ',im3)
+    b = galsim.BoundsI(-K + 1, K, -L + 1, L)
+    im_test = galsim.ImageCD(b, init_value=0)
```

Comment:
what is the purpose of `im_test`? it doesn't seem like you test anything with it

Reply:
Good catch. Hold over from the other test!